### ISSUE TITLE
SVT:User need better message when adding template repo with yaml file extension 

### ISSUE THREAD

**Issue by jagraj** [eclipse/codewind#2837]
**Codewind version:** 0.11.0
**OS:** CoreOS/OpenShift 4.3

**Che version:** 7.9.3 , CodeReady Workspaces 2.1.1
**IDE extension version:** 0.11.0
**IDE version:** CRW 2.1.1/Che 7.9.3
**Kubernetes cluster:** OpenShift 4.3

**Description:**
When adding Kabanero stacks repo with yaml file extension that we display 400 error to the user and we need to display better error message to the user saying "json" file expecting not "yaml" file.

**Steps to reproduce:**
1. Install Codewind from Eclipse Che/CRW.
2. Add Kabanero repo from Template manager. (https://github.com/kabanero-io/kabanero-stack-hub/releases/download/0.8.1/kabanero-stack-hub-index.yaml).
3. You get message "Error adding new template source: Error: PFE responded with status code 400"

**Workaround:**

**Comment by tetchel** [comment]
Agreed, the user should never have to see a status code. This error message is provided by cwctl which gets it from Portal.

**Comment by jcockbain** [comment]
After https://github.com/eclipse/codewind-installer/pull/431 the error returned shouldn't be the one described above but something like: 

![image](https://user-images.githubusercontent.com/31372187/82441241-8e835e00-9a95-11ea-9e28-20bece90f5c8.png)

Investigating now however why the parameter for me is null.

**Comment by jcockbain** [comment]
@jagraj - the fix has now been merged to PFE. Could you verify when you get a change please?
